convince-my-boss.html

```diff
@@ -174,6 +176,7 @@ <h2 class="small-caps center">Convince My Boss</h2>
 
     <!--Core Javascript Files-->
     <script src="js/jquery.min.js"></script>
+    <script src="js/bootstrap.min.js"></script>
```

Comment:
Can you add all the javascript sources from the below list
```
<script src="js/jquery.min.js"></script>

  <script src="js/bootstrap.min.js"></script>

  <script src="js/jquery.easing.min.js"></script>

  <script src="js/jquery.scrollTo.js"></script>

   <script src="js/wow.min.js"></script>

    <script src="js/custom.js"></script>

   <script src="js/jtline.js"></script>

   <script src="js/canvas.js"></script>
```

Reply:
Yes, I have added all js files. Just to let you know that `custom/css/reset.css` and `custom/css/default.css` are not there.

Reply:
see 6401947

Reply:
Uh they are already there https://github.com/pythonindia/inpycon2018/blob/master/convince-my-boss.html#L29-L30